### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by iwolfisberg**
### 💬 Describe the pull request/code changes
Update pull_request_template.md

### 🎟️ Jira
?

### 🔢 To Review
_Steps to review the PR:_
1. Check the template
2. Agree

### CODE REVIEW

**Review by nelsonreitz — APPROVED** [review]
agree

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,21 +1,9 @@
-### 💬 Describe the pull request/code changes
-_A clear and concise description of what the pull request is about._
+### 💬 Description
 
 ### 🎟️ Jira
```
Comment (Gurakh) [inline .github/pull_request_template.md]:
```suggestion
### 🎟️ Issue(s)
```

**Review by Gurakh — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Yago) [reply .github/pull_request_template.md]: “Issues” ?

**Review by Yago — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Seb-GRAF) [reply .github/pull_request_template.md]: "issue(s)"

**Review by Seb-GRAF — APPROVED** [review]

**Review by Gurakh — APPROVED** [review]

**Review by gido — APPROVED** [review]

**Pull request merged**